common/src/autogluon/common/utils/utils.py

```diff
@@ -23,23 +23,28 @@
 
 
 def setup_outputdir(path, warn_if_exist=True, create_dir=True, path_suffix=None):
+    is_s3_path = False
     if path:
-        assert isinstance(
-            path, (str, Path)
-        ), f"Only str and pathlib.Path types are supported for path, got {path} of type {type(path)}."
+        assert isinstance(path, (str, Path)), (
+            f"Only str and pathlib.Path types are supported for path, got {path} of type {type(path)}."
+        )
+
+        is_s3_path = str(path).lower().startswith("s3://")
```

Comment:
why `.lower()`?

Reply:
This is just to incorporate cases when people use capitalized `S3` in the path.